modules/custom/az_migration/migrations/az_node_event.yml

```diff
@@ -64,23 +64,27 @@ process:
         plugin: str_replace
         search: '[node:url]'
         replace: ''
+        source: url
       title:
         plugin: default_value
         source: title
         default_value: 'View Event'
       attributes: attributes
 
   field_az_link:
-    plugin: field_link
-    source: '@preprocess_field_uaqs_link'
-    uri_scheme: "https://"
-
-  'field_az_link/0/uri':
-    -
-      plugin: str_replace
-      source: field_az_link/0/uri
-      search: 'route:<nolink>'
-      replace: ''
+    - plugin: field_link
+      source: '@preprocess_field_uaqs_link'
+      uri_scheme: "https://"
+    - plugin: sub_process
+      process:
+        delta: delta
+        uri:
+          - plugin: str_replace
+            source: uri
+            search: 'route:<nolink>'
+            replace: ''
+        title: title
+        attributes: attributes
```

Comment:
The `sub_process` here is actually using the destination field's values, and not the source field's since this is a chained process, and there is no source set.